### ISSUE TITLE
How do I train a new model on other dataset?

### ISSUE THREAD

**Issue by a2108870** [mahmoudnafifi/SIIE#2]
How do I train a new model on other dataset?(such as INTEL-TAU)

**Comment by mahmoudnafifi** [comment]
Thanks for your question. The main idea of the [paper](https://arxiv.org/abs/1912.06888) is to have a sensor-independent model that does not require retraining for new cameras. You can use the pre-trained models to test on new datasets (such as INTEL-TAU). I did this evaluation on INTEL-TAU and the results are reported [here](https://arxiv.org/abs/2011.11890). 

With that said, of course, the accuracy of SIIE is expected to get improved if the model is trained/tuned on the target dataset. Currently, we are not planning to release the training code. If anything changes, I will post here to get a notification.

**Comment by a2108870** [comment]
Thank you for your answer. I see that you have given results for Quasi-Unsupervised CC and SIIE testing in INTEL-TAU in your new paper, are both methods trained on other datasets and tested in INTEL-TAU?

**Comment by mahmoudnafifi** [comment]
Yes, as both methods designed to be sensor-independent.

**Comment by a2108870** [comment]
Thank you for your patient answer and wish you all the best in your work

**Comment by mahmoudnafifi** [comment]
Thank you so much!

**Comment by mahmoudnafifi** [comment]
A PyTorch implementation is now available [here](https://colab.research.google.com/drive/1EtYszVLgTYv5TZFSu5_y4sCWi6UMCT8Z?usp=sharing). It includes training code.

**Comment by a2108870** [comment]
That's great. Thank you for your work.
